### PR TITLE
bump version to 0.25.1

### PR DESCRIPTION
**Pull request by apoelstra**

### CODE REVIEW

**Comment by apoelstra** [comment]
need to add https://github.com/rust-bitcoin/rust-bitcoin/pull/480 to changelog

**Comment by dr-orlovsky** [comment]
https://github.com/rust-bitcoin/rust-bitcoin/commit/3618d7a41d8aadd1f3c1762969893012cefff899 was a breaking change, so it can't be 0.25.1 anymore. See https://github.com/rust-bitcoin/rust-miniscript/pull/153

**Review by dr-orlovsky — CHANGES_REQUESTED** [review]
NACK: https://github.com/rust-bitcoin/rust-bitcoin/commit/3618d7a41d8aadd1f3c1762969893012cefff899 was a breaking change, so it can't be 0.25.1 anymore. See https://github.com/rust-bitcoin/rust-miniscript/pull/153

**Comment by stevenroose** [comment]
> NACK: [3618d7a](https://github.com/rust-bitcoin/rust-bitcoin/commit/3618d7a41d8aadd1f3c1762969893012cefff899) was a breaking change, so it can't be 0.25.1 anymore. See [rust-bitcoin/rust-miniscript#153](https://github.com/rust-bitcoin/rust-miniscript/pull/153)

Hmm that method returns a `Result` while it shouldn't do that. Engines don't error on IO. I'll create a PR to fix that.

**Comment by stevenroose** [comment]
Here: https://github.com/rust-bitcoin/rust-bitcoin/pull/493

**Comment by apoelstra** [comment]
Ok #493 is merged

**Comment by dr-orlovsky** [comment]
Is it possible to release this version on Monday pls? Any more reviewers, @stevenroose, @elichai?

**Comment by apoelstra** [comment]
@dr-orlovsky I would like to get #413 in first.

**Comment by apoelstra** [comment]
Should maybe add https://github.com/rust-bitcoin/rust-bitcoin/pull/500 to changelog

**Comment by apoelstra** [comment]
Should add https://github.com/rust-bitcoin/rust-bitcoin/pull/413 to changelog

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,14 @@
 
+# 0.25.1 - 2020-10-07
+
+- [Expose methods on `Script`](https://github.com/rust-bitcoin/rust-bitcoin/pull/387) to generate various scriptpubkeys
+- [Expose all cargo features of secp256k1](https://github.com/rust-bitcoin/rust-bitcoin/pull/486)
+- Allow directly creating [various hash newtypes](https://github.com/rust-bitcoin/rust-bitcoin/pull/388)
+- Add methods to `Block` [to get the coinbase tx and BIP34 height commitment](https://github.com/rust-bitcoin/rust-bitcoin/pull/444)
+- [Add `extend` method](https://github.com/rust-bitcoin/rust-bitcoin/pull/459) to bip32::DerivationPath
+- [Alias `(Fingerprint, DerivationPath)` as `KeySource`](https://github.com/rust-bitcoin/rust-bitcoin/pull/480)
+- Expose [the raw sighash message](https://github.com/rust-bitcoin/rust-bitcoin/pull/485) from sighash computations
```
Comment (dr-orlovsky) [inline CHANGELOG.md:10]:
```suggestion
- [Add serde implementation for PSBT data structs](https://github.com/rust-bitcoin/rust-bitcoin/pull/497)
- [Add FromStr/Display implementation for SigHashType](https://github.com/rust-bitcoin/rust-bitcoin/pull/497/commits/a4a7035a947998c8d0d69dab206e97253fd8e048)
- Expose [the raw sighash message](https://github.com/rust-bitcoin/rust-bitcoin/pull/485) from sighash computations
```

**Review by dr-orlovsky — APPROVED** [review]

**Review by TheBlueMatt — APPROVED** [review]
I didnt review anything in the bump, but, like, the version number change looks like a version number change.

**Review by stevenroose — APPROVED** [review]

**Pull request merged**